### DISCUSSION TITLE
Help me using json for better formatting

### DISCUSSION THREAD

**Discussion by ghost** [nlohmann/json#2430] [Q&A]
Hi guys, 

I need to use json for a personnal project, and since I'm using a json, the formatting with "\n" is important to understand better a complex json. 
The thing is when I convert values to string, then back to json, all "\n" disapear, too bad. Is there something I can do to keep a json formatting in a .txt file, so the results can look better?

**Comment by calben** [comment]
Code sample?
Your text is a little unclear for me.

**Comment by nlohmann** [comment]
@callmemaybe75 Can you clarify what you provide an example?
